keep/providers/grafana_incident_provider/grafana_incident_provider.py

```
@@ -158,15 +171,16 @@ def _get_alerts(self) -> list[AlertDto]:
                     incidentMembershipPreview=incident["incidentMembershipPreview"],
                     fieldValues=incident["fieldValues"],
                     version=incident["version"],
-                    source=["grafana_incident"]
+                    source=["grafana_incident"],
                 )
                 for incident in response.json()["incidentPreviews"]
             ]
 
         except Exception as e:
             self.logger.error(f"Failed to get incidents from grafana incident: {e}")
             raise Exception(f"Failed to get incidents from grafana incident: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)